Tests/test_image_resample.py

```diff
@@ -458,6 +459,86 @@ def test_formats(self):
                 cropped = im.crop(box).resize((32, 32), resample)
                 self.assert_image_similar(cropped, with_box, 0.4)
 
+    def test_passthrough(self):
+        "When no resize is required"
```

Comment:
Is there a reason for these to be unused strings and not docstrings or comments? 

Docstrings print out during test run, if I remember, so comments would be better for consistency.

Reply:
Thanks. [Fixed](https://github.com/python-pillow/Pillow/commit/a7cf91499a65978509a7e341c3215444a1c05777)